docs/DeveloperGuide.adoc

```diff
@@ -476,6 +476,7 @@ e.g. `seedu.address.logic.LogicManagerTest`
 * Solution: Execute Gradle task `processResources`.
 
 == Dev Ops
+You need to ensure the next few sections are fulfilled so that the project is fully integrated, tested and built for release. You have to be fully aware of the configurations and the infrastructure of the application and the environment of where the application will be installed.
```

Comment:
`The following sections help you ensure that the project is fully integrated, tested, and built for release.` sounds better. 2nd sentence doesn't seem necessary.

Reply:
Ok thanks!